cpp/src/barretenberg/plonk/proof_system/prover/c_bind.cpp

```diff
@@ -1,13 +1,17 @@
 #include "prover.hpp"
 #include "barretenberg/env/data_store.hpp"
 #include "barretenberg/env/crs.hpp"
+#include "barretenberg/serialize/test_helper.hpp"
+#include "barretenberg/serialize/raw_pointer.hpp"
 
-#define WASM_EXPORT __attribute__((visibility("default")))
+#define WASM_EXPORT extern "C" __attribute__((visibility("default")))
```

Comment:
Note it's better to have extern C in our export declare than have extern C blocks IMO